pkgs/development/compilers/cudatoolkit/redist/overrides.nix

```diff
@@ -20,6 +20,27 @@ in (lib.filterAttrs (attr: _: (prev ? "${attr}")) {
     prev.libcublas
   ];
 
+  cuda_nvcc = prev.cuda_nvcc.overrideAttrs (oldAttrs:
+    let
+      inherit (prev.cudatoolkit) cc;
+    in
+    {
+      postInstall = (oldAttrs.postInstall or "") + ''
+        # Point NVCC at a compatible compiler
+        # FIXME: non-redist cudatoolkit copy-pastes this code
```

Comment:
let's try to keep comments out of shell strings and in nix as much as possible. otherwise drv hashes change every time we change comments